src/win/core.c

```diff
@@ -98,7 +95,12 @@ static int uv__loops_add(uv_loop_t* loop) {
 
   uv_mutex_lock(&uv__loops_lock);
 
-  if (uv__loops_size == uv__loops_capacity) {
+  if (uv__loops == NULL) {
```

Comment:
You can drop this block if you ensure `uv__loops_capacity == 0` when `uv__loops == NULL`.